src/modules/vtol_att_control/tailsitter.cpp

```diff
@@ -157,7 +428,7 @@ void Tailsitter::fill_actuator_outputs()
 			} else {
 				// NOTE: There is no mistake in the line below, multicopter yaw axis is controlled by elevon roll actuation!
 				_actuators_out_1->control[actuator_controls_s::INDEX_ROLL] = _actuators_mc_in->control[actuator_controls_s::INDEX_YAW];	//roll elevon
-				_actuators_out_1->control[actuator_controls_s::INDEX_PITCH] = _actuators_mc_in->control[actuator_controls_s::INDEX_PITCH];	//pitch elevon
+				_actuators_out_1->control[actuator_controls_s::INDEX_PITCH] =0;	//pitch elevon
```

Comment:
@davidvor We cannot hardcode this since e.g. the duorotor tailsitter needs the elevons for controlling pitch in mc mode.